Libraries/test/Amazon.Lambda.Annotations.SourceGenerators.Tests/Snapshots/Functions_ToUpper_Generated_NET8.g.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using System.Linq;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using System.IO;
+using Amazon.Lambda.Core;
+
+namespace TestServerlessApp.NET8
+{
+    public class Functions_ToUpper_Generated
+    {
+        private readonly Functions functions;
+        private readonly Amazon.Lambda.Serialization.SystemTextJson.DefaultLambdaJsonSerializer serializer;
+
+        public Functions_ToUpper_Generated()
+        {
+            SetExecutionEnvironment();
+            functions = new Functions();
+            serializer = new Amazon.Lambda.Serialization.SystemTextJson.DefaultLambdaJsonSerializer();
+        }
+
+        public string ToUpper(string text)
+        {
+            return functions.ToUpper(text);
+        }
+
+        private static void SetExecutionEnvironment()
+        {
+            const string envName = "AWS_EXECUTION_ENV";
+
+            var envValue = new StringBuilder();
+
+            // If there is an existing execution environment variable add the annotations package as a suffix.
+            if(!string.IsNullOrEmpty(Environment.GetEnvironmentVariable(envName)))
+            {
+                envValue.Append($"{Environment.GetEnvironmentVariable(envName)}_");
+            }
+
+            envValue.Append("amazon-lambda-annotations_1.1.0.0");
```

Comment:
Should we just increment the version to the one we intend to release this PR in? (and do it for all the other snaphsot files)

Reply:
Given the churn we've had with .NET 8, I didn't want to tie this to a version yet so would just do the separate versioning PR.